deployments/helm/nvidia-device-plugin/templates/daemonset-device-plugin.yml

```diff
@@ -38,6 +38,7 @@ spec:
     metadata:
       labels:
         {{- include "nvidia-device-plugin.templateLabels" . | nindent 8 }}
+        {{- toYaml .Values.podLabels | nindent 8 }}
```

Comment:
A general question. What is the behaviour if these conflict with any labels generated from `templateLabels`?

Reply:
It should merge and keep the last stated label I believe.

Reply:
So if someone passed in a label the templateLabels variable sets, they will override it.